lib/install/install.sh

```diff
@@ -384,14 +384,14 @@ bpkg_install_from_remote () {
     fi
 
     if [ -z "$PREFIX" ]; then
-      if [ $USER == 'root' ]; then
+      if [[ $USER == "root" ]]; then
```

Comment:
Shouldn't this have double quotes for `$USER` (as it should be parsed) and single quotes for `root` (as it need not be parsed)? If this is done, then there is no need to switch to the UNIX incompatible `[[` but `[` should just work.

i.e:
```
 if [ "$USER" == 'root' ]; then
```

Reference: https://stackoverflow.com/questions/13617843/

Reply:
fixed!